btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/service/BtcRollbackService.kt

```diff
@@ -41,8 +50,22 @@ class BtcRollbackService(
             withdrawalDetails.withdrawalTime,
             withdrawalDetails.toAddress
         )
-
-        rollbackService.rollback(withdrawalFinalizationDetails, reason).fold(
+        withdrawalConsumer.getConsumerQuorum().flatMap { quorum ->
+            var transaction = rollbackService.createRollbackTransaction(withdrawalFinalizationDetails, reason, quorum)
+            if (btcTx != null) {
+                logger.info("Rollback Bitcoin transaction as well $btcTx")
+                val withdrawalRollbackData = WithdrawalRollbackData(
+                    withdrawalDetails,
+                    btcTx.inputs.map { input -> input.irohaKey() })
+                transaction = transaction.makeMutable()
```

Comment:
does `createRollbackTransaction` returns a signed transaction?

Reply:
No. It's unsigned.